scripts/servers-build.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -e -x -E
+
+
+echo "WORKSPACE=$WORKSPACE"
+
+# ---------------------------------------------------
+# Preliminary checks.
+# ---------------------------------------------------
+
+# Verify that version references are up to date.
+$WORKSPACE/integration/extra/release_tool.py --verify-integration-references
+
+# ---------------------------------------------------
+# Build server repositories.
+# ---------------------------------------------------
+
+if grep mender_servers <<<"$JOB_BASE_NAME"; then
+    # Use release tool to query for available docker names.
+    for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker ); do (
+
+        git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
+        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+
+        case "$docker" in
+            deployments|deployments-enterprise|deviceauth|inventory|inventory-enterprise|tenantadm|useradm|useradm-enterprise|workflows|workflows-enterprise|workflows-worker|workflows-enterprise-worker|create-artifact-worker|auditlogs|mtls-ambassador|deviceconnect)
+                cd go/src/github.com/mendersoftware/$git
```

Comment:
Suggestion:
For future additions at the end here, should this be run in a sub-shell? :)

Reply:
We are already in a subshell (see `(` in` for docker in...`). When could we possibly need another one here?

Reply:
Oh, I see. Up to you then. Base security net is already there :+1: